scrapers_next/mt/committees.py

```diff
@@ -0,0 +1,228 @@
+from spatula import (
+    HtmlPage,
+    HtmlListPage,
+    XPath,
+    SelectorError,
+    URL,
+    SkipItem,
+)
+from openstates.models import ScrapeCommittee
+import logging
+import re
+import urllib.parse  # Need to urlencode a string
+
+
+# This page contains information on every committee, except for administrative
+# committees, which are handled by AdministrativeCommitteeList
+class HouseSenateJointCommList(HtmlListPage):
+    source = "http://laws.leg.mt.gov/legprd/law0240w$cmte.startup"
+
+    # not(text()=" ") skips the blank option
+    selector = XPath("//select[@name='P_COM_NM']/option[not(text()=' ')]/text()")
+
+    def process_item(self, committee_title):
+
+        # Need to get the latest session, it's the one that's pre-selected
+        session = (
+            XPath("//select[@name='P_SESS']/option[@selected='selected']")
+            .match_one(self.root)
+            .get("value")
+        )
+
+        # To build the member info url, special characters must be escaped
+        committee_url_name = urllib.parse.quote_plus(committee_title)
+        member_info_href = f"http://laws.leg.mt.gov/legprd/LAW0240W$CMTE.ActionQuery?P_SESS={session}&P_COM_NM={committee_url_name}&P_ACTN_DTM=&U_ACTN_DTM=&Z_ACTION2=Find"
+
+        # Committee names may have abbreviations, replace them with the full word
+        committee_title = committee_title.replace(
+            "Approps ", "Appropriations "
+        ).replace("Subcom ", "Subcommittee ")
+
+        chamber = None
+        parent = None
+        classification = "committee"
+
+        # These next steps will remove parts of the title
+        # Each step builds on the last, so their order is important
+
+        # 1) Determine the chamber from the title prefix
+        house_prefix = "(H) "
+        senate_prefix = "(S) "
+        if committee_title.startswith(house_prefix):
+            chamber = "lower"
+            committee_title = committee_title[len(house_prefix) :]
+        if committee_title.startswith(senate_prefix):
+            chamber = "upper"
+            committee_title = committee_title[len(senate_prefix) :]
+
+        # 2) A joint prefix may exist
+        joint_prefix = "Joint "
+        if committee_title.startswith(joint_prefix):
+            chamber = "legislature"
+            committee_title = committee_title[len(joint_prefix) :]
```

Comment:
Remove this (and adjust the code and comment numbering accordingly). Apologies, but it's recently come to my attention that we want to keep any occurrences of either "Joint" or "Legislative" substring at the beginning of a committee name.

Reply:
Fixed with https://github.com/openstates/openstates-scrapers/pull/4451/commits/9c05a8d545d8990e651dc3449aed11c3085e17be and https://github.com/openstates/openstates-scrapers/pull/4451/commits/5864afd567afd470841ee195a6f3f5d6d21743bb